Tools/Frame_params/Solo_AC36.param

```diff
@@ -0,0 +1,133 @@
+# NOTE: 1/8/2018: Required parameters for a 3DR Solo with ArduCopter 3.6. Not for use on ArduCopter 3.5!
+ACCEL_Z_I,1.5
+ACCEL_Z_P,0.75
+ACRO_RP_P,4
+ACRO_THR_MID,0.4
+ACRO_YAW_P,1
+AHRS_ORIENTATION,12
+ANGLE_MAX,2000
+ARMING_CHECK,7166
+ATC_ACCEL_P_MAX,36000
+ATC_ACCEL_R_MAX,36000
+ATC_ACCEL_Y_MAX,7000
+ATC_ANG_PIT_P,10
+ATC_ANG_RLL_P,9
+ATC_ANG_YAW_P,6
+ATC_RAT_PIT_D,0.006
+ATC_RAT_PIT_IMAX,0.444
+ATC_RAT_PIT_P,0.167
+ATC_RAT_RLL_D,0.005
+ATC_RAT_RLL_I,0.12
+ATC_RAT_RLL_IMAX,0.444
+ATC_RAT_RLL_P,0.086
+ATC_RAT_YAW_FILT,5
+ATC_RAT_YAW_I,0.05
+ATC_RAT_YAW_IMAX,0.222
+ATC_RAT_YAW_P,0.5
+ATC_SLEW_YAW,8000
+BATT_CAPACITY,5200
+BATT_MONITOR,5
+BATT_SERIAL_NUM,0
+BRD_IMU_TARGTEMP,45
+BRD_SAFETY_MASK,16368
+BRD_SAFETYENABLE,0
+BRD_SER1_RTSCTS,1
+CIRCLE_RADIUS,3000
+COMPASS_ORIENT,38
+COMPASS_ORIENT2,0
+COMPASS_ORIENT3,0
+COMPASS_PRIMARY,0
+COMPASS_TYPEMASK,15392
+COMPASS_USE,1
+COMPASS_USE2,1
+COMPASS_USE3,0
+DISARM_DELAY,8
+FENCE_ACTION,0
+FENCE_ALT_MAX,122
```

Comment:
The pixhawk default is 100.
Does it mean that it is changing intentionally?

Reply:
Solo uses 122, which is 400FT.  Yes, intentional.